roles/openshift_master/defaults/main.yml

```diff
@@ -29,4 +29,4 @@ os_firewall_deny:
 - service: former etcd peer port
   port: 7001/tcp
 
-openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
openshift_pkg_version and it's containerized counterpart openshift_image_tag are optional overrides.  They won't be set normally.